mysql-test/suite/encryption/r/innodb_encryption_aborted_rotation.result

```diff
@@ -76,8 +76,10 @@ SET GLOBAL debug="+d,crash_on_t1_flush_after_dd_update";
 SET GLOBAL default_table_encryption=ONLINE_FROM_KEYRING_TO_UNENCRYPTED;
 # Make sure that t1 has been decrypted
 # First restart the server after crash so any redo logs for t1 were proceed. If there are any logs available
-# t1 will be opened without validating idb file to DD.
-# restart:--innodb-encryption-rotate-key-age=15 --innodb-encryption-threads=0
+# t1 will be opened without validating idb file to DD. We set here innodb-encryption-threads to value > 0 - so
+# in case there is no redo for t1 and validation to DD would occur we would not get an error in the log
+# to set innodb-encryption-threads to value > 0 to finish rotation. We grep for this error later.
```

Comment:
Also see if you want to use use log_buffer_flush_to_disk(); just before the crash. This will ensure the redo is always on disk for the test scenario

Reply:
Ok, it looks like we want the update to DD reach the actual data page on mysql.ibd, not just redo? correct?

Reply:
Yes

Reply:
Ok, then  buf_LRU_flush_or_remove_pages(space_id, BUF_REMOVE_FLUSH_WRITE, nullptr,
                                      false);  before crash could work.
What do you think?

Reply:
Yes, I think it should work. I just think that if it possible to obtain the same in the MTR test, it is better not to add extra code with some debug statement. If you do not mind, I would like to keep it this way.

Reply:
For consistency reasons, it is perfectly fine to add code before a debug crash. See code like this

/* Test what happens on crash here.
      The data dictionary transaction should be
      rolled back, restoring the old table. */
      DBUG_EXECUTE_IF("innodb_alter_commit_crash_before_commit",
                      log_buffer_flush_to_disk();
                      DBUG_SUICIDE(););
      ut_ad(!trx->fts_trx);
  
      DBUG_EXECUTE_IF("innodb_alter_commit_crash_after_commit",
                      log_make_latest_checkpoint();
                      log_buffer_flush_to_disk(); DBUG_SUICIDE(););

 DBUG_EXECUTE_IF("row_mysql_crash_if_error", {
          log_buffer_flush_to_disk();
          DBUG_SUICIDE();
        });

For this time, ok to skip..